test/errorInfoTests.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+import { validateExamples } from "../lib/validate"
+import * as assert from "assert"
+
+describe("error info", () => {
+  it("error info should contain url to file", async () => {
```

Comment:
it'd be great to indicate in the test, maybe with a comment, what is special to this swagger/example combination that is helping validate the functionality, since that's generally the key to these tests.